.github/workflows/build-package.yml

```diff
@@ -36,12 +33,6 @@ jobs:
 
       - name: configure-node
         uses: actions/setup-node@v4
-        with:
-          node-version: 20
```

Comment:
Does it use node version 20 by default?

Reply:
Hm, I feel more comfortable with it being explicit, rather than defaulted.

Reply:
```diff
      - name: configure-node
        uses: actions/setup-node@v4
+        with:
+          node-version: 20
```